// faceCentering/test/testbeds/eventAndImagePack.js
/*globals module*/

var r292x326Image0 = {
    event: {"data": [{"total": 2, "width": 90, "height": 90, "x": 43, "y": 13}]},
    imagePack: {
        "width": 292,
        "height": 326,
        "imageClassName": ".cm_mobHeader_artist-image--style-0-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/386998_512092287718_750833973_n.jpg"
    }
};
var r166x250Image1 = {
    event: {"data": [{"total": 5, "width": 43, "height": 43, "x": 51, "y": 45}]},
    imagePack: {
        "width": 166,
        "height": 250,
        "imageClassName": ".cm_mobHeader_artist-image--style-1-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/88e34c17bf76c1d5178b0ce08d9934e6.jpg"
    }
};
var r400x268Image3 = {
    event: {"data": [{"total": 1, "width": 39, "height": 39, "x": 177, "y": 21}]},
    imagePack: {
        "width": 400,
        "height": 268,
        "imageClassName": ".cm_mobHeader_artist-image--style-3-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001325433.jpg"
    }
};
var r400x272Image5 = {
    event: {
        "data": [{"total": 3, "width": 39, "height": 39, "x": 25, "y": 56}, {
            "total": 1,
            "width": 39,
            "height": 39,
            "x": 177,
            "y": 84
        }, {"total": 6, "width": 46, "height": 46, "x": 293, "y": 29}]
    },
    imagePack: {
        "width": 400,
        "height": 272,
        "imageClassName": ".cm_mobHeader_artist-image--style-5-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001327874.jpg"
    }
};
var r250x206Image4 = {
    event: {
        "data": [{"total": 4, "width": 39, "height": 39, "x": 102, "y": 17}, {
            "total": 2,
            "width": 39,
            "height": 39,
            "x": 27,
            "y": 41
        }]
    },
    imagePack: {
        "width": 250,
        "height": 206,
        "imageClassName": ".cm_mobHeader_artist-image--style-4-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001326338.jpg"
    }
};
var r250x248Image7 = {
    event: {"data": [{"total": 1, "width": 76, "height": 76, "x": 90, "y": 42}]},
    imagePack: {
        "width": 250,
        "height": 248,
        "imageClassName": ".cm_mobHeader_artist-image--style-7-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001328037.jpg"
    }
};
var r400x251Image6 = {
    event: {
        "data": [{"total": 3, "width": 39, "height": 39, "x": 271, "y": 119}, {
            "total": 1,
            "width": 61,
            "height": 61,
            "x": 330,
            "y": 80
        }, {"total": 3, "width": 46, "height": 46, "x": 18, "y": 96}, {
            "total": 2,
            "width": 107,
            "height": 107,
            "x": 88,
            "y": 75
        }]
    },
    imagePack: {
        "width": 400,
        "height": 251,
        "imageClassName": ".cm_mobHeader_artist-image--style-6-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001327893.jpg"
    }
};
var r400x268Image8 = {
    event: {"data": [{"total": 1, "width": 95, "height": 95, "x": 112, "y": 80}]},
    imagePack: {
        "width": 400,
        "height": 268,
        "imageClassName": ".cm_mobHeader_artist-image--style-8-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001329528.jpg"
    }
};
var r400x302Image9 = {
    event: {
        "data": [{"total": 3, "width": 39, "height": 39, "x": 174, "y": 96}, {
            "total": 3,
            "width": 39,
            "height": 39,
            "x": 44,
            "y": 116
        }, {"total": 3, "width": 39, "height": 39, "x": 322, "y": 170}]
    },
    imagePack: {
        "width": 400,
        "height": 302,
        "imageClassName": ".cm_mobHeader_artist-image--style-9-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001329620.jpg"
    }
};
var r400x284Image11 = {
    event: {
        "data": [{"total": 4, "width": 45, "height": 45, "x": 176, "y": 18}, {
            "total": 3,
            "width": 54,
            "height": 54,
            "x": 235,
            "y": 33
        }, {"total": 1, "width": 61, "height": 61, "x": 125, "y": 40}, {
            "total": 5,
            "width": 67,
            "height": 67,
            "x": 295,
            "y": 21
        }, {"total": 7, "width": 62, "height": 62, "x": 28, "y": 41}]
    },
    imagePack: {
        "width": 400,
        "height": 284,
        "imageClassName": ".cm_mobHeader_artist-image--style-11-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001345160.jpg"
    }
};
var r400x309Image13 = {
    event: {
        "data": [{"total": 3, "width": 54, "height": 54, "x": 133, "y": 89}, {
            "total": 5,
            "width": 64,
            "height": 64,
            "x": 236,
            "y": 31
        }, {"total": 6, "width": 66, "height": 66, "x": 15, "y": 70}]
    },
    imagePack: {
        "width": 400,
        "height": 309,
        "imageClassName": ".cm_mobHeader_artist-image--style-13-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001366608.jpg"
    }
};
var r250x245Image14 = {
    event: {
        "data": [{"total": 1, "width": 39, "height": 39, "x": 105, "y": 102}, {
            "total": 2,
            "width": 39,
            "height": 39,
            "x": 176,
            "y": 123
        }]
    },
    imagePack: {
        "width": 250,
        "height": 245,
        "imageClassName": ".cm_mobHeader_artist-image--style-14-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001395010.jpg"
    }
};
var r400x276Image17 = {
    event: {"data": [{"total": 1, "width": 61, "height": 61, "x": 245, "y": 35}]},
    imagePack: {
        "width": 400,
        "height": 276,
        "imageClassName": ".cm_mobHeader_artist-image--style-17-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001399345.jpg"
    }
};
var r400x263Image18 = {
    event: {
        "data": [{"total": 2, "width": 61, "height": 61, "x": 318, "y": 33}, {
            "total": 3,
            "width": 61,
            "height": 61,
            "x": 198,
            "y": 47
        }, {"total": 2, "width": 69, "height": 69, "x": 16, "y": 76}, {
            "total": 8,
            "width": 70,
            "height": 70,
            "x": 117,
            "y": 61
        }]
    },
    imagePack: {
        "width": 400,
        "height": 263,
        "imageClassName": ".cm_mobHeader_artist-image--style-18-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001401588.jpg"
    }
};
var r400x265Image19 = {
    event: {
        "data": [{"total": 4, "width": 39, "height": 39, "x": 302, "y": 32}, {
            "total": 3,
            "width": 39,
            "height": 39,
            "x": 101,
            "y": 40
        }, {"total": 3, "width": 39, "height": 39, "x": 200, "y": 46}, {
            "total": 1,
            "width": 39,
            "height": 39,
            "x": 63,
            "y": 60
        }, {"total": 5, "width": 48, "height": 48, "x": 159, "y": 124}]
    },
    imagePack: {
        "width": 400,
        "height": 265,
        "imageClassName": ".cm_mobHeader_artist-image--style-19-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001401684.jpg"
    }
};
var r357x400Image20 = {
    event: {"data": [{"total": 8, "width": 139, "height": 139, "x": 86, "y": 66}]},
    imagePack: {
        "width": 357,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-20-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001402140.jpg"
    }
};
var r400x251Image21 = {
    event: {
        "data": [{"total": 2, "width": 39, "height": 39, "x": 119, "y": 30}, {
            "total": 4,
            "width": 39,
            "height": 39,
            "x": 306,
            "y": 31
        }, {"total": 4, "width": 39, "height": 39, "x": 180, "y": 37}, {
            "total": 4,
            "width": 39,
            "height": 39,
            "x": 245,
            "y": 44
        }]
    },
    imagePack: {
        "width": 400,
        "height": 251,
        "imageClassName": ".cm_mobHeader_artist-image--style-21-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001404843.jpg"
    }
};
var r400x215Image22 = {
    event: {
        "data": [{"total": 3, "width": 39, "height": 39, "x": 100, "y": 13}, {
            "total": 6,
            "width": 87,
            "height": 87,
            "x": 204,
            "y": 73
        }]
    },
    imagePack: {
        "width": 400,
        "height": 215,
        "imageClassName": ".cm_mobHeader_artist-image--style-22-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001405078.jpg"
    }
};
var r250x245Image25 = {
    event: {"data": [{"total": 1, "width": 39, "height": 39, "x": 114, "y": 165}]},
    imagePack: {
        "width": 250,
        "height": 245,
        "imageClassName": ".cm_mobHeader_artist-image--style-25-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001413988.jpg"
    }
};
var r250x202Image26 = {
    event: {
        "data": [{"total": 1, "width": 61, "height": 61, "x": 45, "y": 30}, {
            "total": 1,
            "width": 61,
            "height": 61,
            "x": 185,
            "y": 40
        }]
    },
    imagePack: {
        "width": 250,
        "height": 202,
        "imageClassName": ".cm_mobHeader_artist-image--style-26-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001418164.jpg"
    }
};
var r400x269Image28 = {
    event: {"data": [{"total": 3, "width": 54, "height": 54, "x": 144, "y": 86}]},
    imagePack: {
        "width": 400,
        "height": 269,
        "imageClassName": ".cm_mobHeader_artist-image--style-28-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0002749679.jpg"
    }
};
var r250x209Image27 = {
    event: {"data": [{"total": 2, "width": 39, "height": 39, "x": 35, "y": 21}]},
    imagePack: {
        "width": 250,
        "height": 209,
        "imageClassName": ".cm_mobHeader_artist-image--style-27-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0001465413.jpg"
    }
};
var r400x400Image31 = {
    event: {
        "data": [{"total": 1, "width": 39, "height": 39, "x": 48, "y": 135}, {
            "total": 2,
            "width": 39,
            "height": 39,
            "x": 281,
            "y": 147
        }, {"total": 4, "width": 45, "height": 45, "x": 199, "y": 92}]
    },
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-31-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003148875.jpg"
    }
};
var r399x400Image34 = {
    event: {
        "data": [{"total": 1, "width": 39, "height": 39, "x": 102, "y": 90}, {
            "total": 3,
            "width": 39,
            "height": 39,
            "x": 262,
            "y": 109
        }, {"total": 1, "width": 61, "height": 61, "x": 170, "y": 45}]
    },
    imagePack: {
        "width": 399,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-34-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003276341.jpg"
    }
};
var r400x313Image35 = {
    event: {
        "data": [{"total": 6, "width": 66, "height": 66, "x": 106, "y": 82}, {
            "total": 11,
            "width": 78,
            "height": 78,
            "x": 223,
            "y": 49
        }, {"total": 2, "width": 107, "height": 107, "x": 279, "y": 48}]
    },
    imagePack: {
        "width": 400,
        "height": 313,
        "imageClassName": ".cm_mobHeader_artist-image--style-35-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003277352.jpg"
    }
};
var r400x223Image36 = {
    event: {
        "data": [{"total": 1, "width": 61, "height": 61, "x": 100, "y": 155}, {
            "total": 6,
            "width": 64,
            "height": 64,
            "x": 201,
            "y": 52
        }, {"total": 5, "width": 68, "height": 68, "x": 288, "y": 46}, {
            "total": 8,
            "width": 82,
            "height": 82,
            "x": 115,
            "y": 47
        }]
    },
    imagePack: {
        "width": 400,
        "height": 223,
        "imageClassName": ".cm_mobHeader_artist-image--style-36-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003347847.jpg"
    }
};
var r400x214Image38 = {
    event: {"data": [{"total": 1, "width": 61, "height": 61, "x": 290, "y": 40}]},
    imagePack: {
        "width": 400,
        "height": 214,
        "imageClassName": ".cm_mobHeader_artist-image--style-38-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003351373.jpg"
    }
};
var r400x262Image41 = {
    event: {
        "data": [{"total": 1, "width": 61, "height": 61, "x": 140, "y": 100}, {
            "total": 2,
            "width": 61,
            "height": 61,
            "x": 270,
            "y": 103
        }, {"total": 2, "width": 61, "height": 61, "x": 218, "y": 125}, {
            "total": 4,
            "width": 65,
            "height": 65,
            "x": 77,
            "y": 99
        }]
    },
    imagePack: {
        "width": 400,
        "height": 262,
        "imageClassName": ".cm_mobHeader_artist-image--style-41-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003361490.jpg"
    }
};
var r250x248Image43 = {
    event: {
        "data": [{"total": 1, "width": 61, "height": 61, "x": 145, "y": 105}, {
            "total": 4,
            "width": 56,
            "height": 56,
            "x": 46,
            "y": 108
        }]
    },
    imagePack: {
        "width": 250,
        "height": 248,
        "imageClassName": ".cm_mobHeader_artist-image--style-43-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003378772.jpg"
    }
};
var r400x267Image45 = {
    event: {"data": [{"total": 1, "width": 119, "height": 119, "x": 100, "y": 20}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-45-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003421888.jpg"
    }
};
var r250x250Image49 = {
    event: {
        "data": [{"total": 2, "width": 61, "height": 61, "x": 90, "y": 28}, {
            "total": 2,
            "width": 61,
            "height": 61,
            "x": 38,
            "y": 40
        }]
    },
    imagePack: {
        "width": 250,
        "height": 250,
        "imageClassName": ".cm_mobHeader_artist-image--style-49-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003445129.jpg"
    }
};
var r400x267Image50 = {
    event: {
        "data": [{"total": 7, "width": 45, "height": 45, "x": 185, "y": 39}, {
            "total": 6,
            "width": 57,
            "height": 57,
            "x": 293,
            "y": 38
        }, {"total": 5, "width": 48, "height": 48, "x": 59, "y": 65}]
    },
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-50-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003481272.jpg"
    }
};
var r250x205Image52 = {
    event: {
        "data": [{"total": 2, "width": 35, "height": 35, "x": 18, "y": 20}, {
            "total": 2,
            "width": 35,
            "height": 35,
            "x": 183,
            "y": 35
        }]
    },
    imagePack: {
        "width": 250,
        "height": 205,
        "imageClassName": ".cm_mobHeader_artist-image--style-52-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003495398.jpg"
    }
};
var r250x249Image55 = {
    event: {"data": [{"total": 2, "width": 61, "height": 61, "x": 83, "y": 0}]},
    imagePack: {
        "width": 250,
        "height": 249,
        "imageClassName": ".cm_mobHeader_artist-image--style-55-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003571143.jpg"
    }
};
var r250x250Image54 = {
    event: {"data": [{"total": 3, "width": 72, "height": 72, "x": 82, "y": 31}]},
    imagePack: {
        "width": 250,
        "height": 250,
        "imageClassName": ".cm_mobHeader_artist-image--style-54-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003535164.jpg"
    }
};
var r400x267Image56 = {
    event: {"data": [{"total": 2, "width": 39, "height": 39, "x": 107, "y": 36}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-56-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003584762.jpg"
    }
};
var r400x267Image58 = {
    event: {
        "data": [{"total": 4, "width": 39, "height": 39, "x": 209, "y": 29}, {
            "total": 3,
            "width": 39,
            "height": 39,
            "x": 287,
            "y": 28
        }, {"total": 4, "width": 39, "height": 39, "x": 85, "y": 34}, {
            "total": 5,
            "width": 39,
            "height": 39,
            "x": 147,
            "y": 39
        }]
    },
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-58-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003590626.jpg"
    }
};
var r250x229Image59 = {
    event: {"data": [{"total": 2, "width": 50, "height": 50, "x": 57, "y": 13}]},
    imagePack: {
        "width": 250,
        "height": 229,
        "imageClassName": ".cm_mobHeader_artist-image--style-59-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003592855.jpg"
    }
};
var r266x400Image62 = {
    event: {"data": [{"total": 1, "width": 95, "height": 95, "x": 72, "y": 56}]},
    imagePack: {
        "width": 266,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-62-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003627097.jpg"
    }
};
var r250x220Image64 = {
    event: {"data": [{"total": 1, "width": 61, "height": 61, "x": 115, "y": 30}]},
    imagePack: {
        "width": 250,
        "height": 220,
        "imageClassName": ".cm_mobHeader_artist-image--style-64-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003641474.jpg"
    }
};
var r400x266Image65 = {
    event: {
        "data": [{"total": 4, "width": 56, "height": 56, "x": 291, "y": 88}, {
            "total": 2,
            "width": 119,
            "height": 119,
            "x": 140,
            "y": 35
        }]
    },
    imagePack: {
        "width": 400,
        "height": 266,
        "imageClassName": ".cm_mobHeader_artist-image--style-65-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003689189.jpg"
    }
};
var r400x267Image66 = {
    event: {"data": [{"total": 1, "width": 61, "height": 61, "x": 275, "y": 30}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-66-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003709783.jpg"
    }
};
var r400x225Image67 = {
    event: {
        "data": [{"total": 2, "width": 39, "height": 39, "x": 131, "y": 53}, {
            "total": 3,
            "width": 39,
            "height": 39,
            "x": 4,
            "y": 74
        }, {"total": 3, "width": 39, "height": 39, "x": 351, "y": 84}, {
            "total": 6,
            "width": 43,
            "height": 43,
            "x": 276,
            "y": 76
        }]
    },
    imagePack: {
        "width": 400,
        "height": 225,
        "imageClassName": ".cm_mobHeader_artist-image--style-67-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003709999.jpg"
    }
};
var r400x400Image69 = {
    event: {"data": [{"total": 1, "width": 39, "height": 39, "x": 306, "y": 69}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-69-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003720987.jpg"
    }
};
var r400x250Image70 = {
    event: {"data": [{"total": 1, "width": 39, "height": 39, "x": 141, "y": 18}]},
    imagePack: {
        "width": 400,
        "height": 250,
        "imageClassName": ".cm_mobHeader_artist-image--style-70-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003732467.jpg"
    }
};
var r400x400Image71 = {
    event: {
        "data": [{"total": 1, "width": 61, "height": 61, "x": 180, "y": 310}, {
            "total": 1,
            "width": 76,
            "height": 76,
            "x": 156,
            "y": 0
        }, {"total": 2, "width": 107, "height": 107, "x": 57, "y": 26}]
    },
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-71-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003752413.jpg"
    }
};
var r400x400Image75 = {
    event: {"data": [{"total": 3, "width": 129, "height": 129, "x": 129, "y": 86}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-75-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003795324.jpg"
    }
};
var r400x400Image76 = {
    event: {"data": [{"total": 1, "width": 119, "height": 119, "x": 160, "y": 50}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-76-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003795325.jpg"
    }
};
var r400x400Image77 = {
    event: {"data": [{"total": 6, "width": 66, "height": 66, "x": 284, "y": 102}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-77-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003798761.jpg"
    }
};
var r400x400Image78 = {
    event: {"data": [{"total": 4, "width": 56, "height": 56, "x": 98, "y": 102}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-78-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003802021.jpg"
    }
};
var r250x245Image79 = {
    event: {"data": [{"total": 2, "width": 39, "height": 39, "x": 104, "y": 54}]},
    imagePack: {
        "width": 250,
        "height": 245,
        "imageClassName": ".cm_mobHeader_artist-image--style-79-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003803855.jpg"
    }
};
var r400x267Image80 = {
    event: {"data": [{"total": 4, "width": 50, "height": 50, "x": 232, "y": 33}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-80-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003814405.jpg"
    }
};
var r269x400Image81 = {
    event: {"data": [{"total": 3, "width": 39, "height": 39, "x": 110, "y": 46}]},
    imagePack: {
        "width": 269,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-81-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003828479.jpg"
    }
};
var r400x267Image82 = {
    event: {
        "data": [{"total": 5, "width": 39, "height": 39, "x": 218, "y": 20}, {
            "total": 2,
            "width": 39,
            "height": 39,
            "x": 71,
            "y": 51
        }, {"total": 2, "width": 39, "height": 39, "x": 158, "y": 102}, {
            "total": 5,
            "width": 43,
            "height": 43,
            "x": 333,
            "y": 80
        }]
    },
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-82-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003832775.jpg"
    }
};
var r400x400Image84 = {
    event: {
        "data": [{"total": 7, "width": 39, "height": 39, "x": 158, "y": 66}, {
            "total": 2,
            "width": 39,
            "height": 39,
            "x": 107,
            "y": 81
        }, {"total": 1, "width": 61, "height": 61, "x": 320, "y": 75}, {
            "total": 4,
            "width": 59,
            "height": 59,
            "x": 213,
            "y": 77
        }]
    },
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-84-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003835479.jpg"
    }
};
var r400x400Image85 = {
    event: {
        "data": [{"total": 1, "width": 61, "height": 61, "x": 235, "y": 5}, {
            "total": 4,
            "width": 113,
            "height": 113,
            "x": 141,
            "y": 49
        }]
    },
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-85-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003837167.jpg"
    }
};
var r400x267Image86 = {
    event: {
        "data": [{"total": 1, "width": 39, "height": 39, "x": 234, "y": 162}, {
            "total": 3,
            "width": 111,
            "height": 111,
            "x": 205,
            "y": 21
        }]
    },
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-86-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003840183.jpg"
    }
};
var r400x400Image87 = {
    event: {
        "data": [{"total": 1, "width": 39, "height": 39, "x": 309, "y": 117}, {
            "total": 5,
            "width": 48,
            "height": 48,
            "x": 203,
            "y": 111
        }]
    },
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-87-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003840374.jpg"
    }
};
var r400x255Image89 = {
    event: {
        "data": [{"total": 3, "width": 39, "height": 39, "x": 211, "y": 43}, {
            "total": 1,
            "width": 39,
            "height": 39,
            "x": 42,
            "y": 48
        }, {"total": 2, "width": 39, "height": 39, "x": 113, "y": 57}, {
            "total": 3,
            "width": 39,
            "height": 39,
            "x": 319,
            "y": 55
        }, {"total": 3, "width": 39, "height": 39, "x": 253, "y": 64}, {
            "total": 3,
            "width": 39,
            "height": 39,
            "x": 164,
            "y": 74
        }]
    },
    imagePack: {
        "width": 400,
        "height": 255,
        "imageClassName": ".cm_mobHeader_artist-image--style-89-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003842964.jpg"
    }
};
var r400x400Image90 = {
    event: {"data": [{"total": 1, "width": 76, "height": 76, "x": 162, "y": 36}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-90-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003848198.jpg"
    }
};
var r400x400Image93 = {
    event: {"data": [{"total": 1, "width": 61, "height": 61, "x": 200, "y": 80}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-93-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003859455.jpg"
    }
};
var r400x400Image94 = {
    event: {"data": [{"total": 1, "width": 119, "height": 119, "x": 140, "y": 110}]},
    imagePack: {
        "width": 400,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-94-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003867863.jpg"
    }
};
var r400x202Image96 = {
    event: {
        "data": [{"total": 3, "width": 39, "height": 39, "x": 126, "y": 36}, {
            "total": 1,
            "width": 39,
            "height": 39,
            "x": 213,
            "y": 48
        }, {"total": 2, "width": 39, "height": 39, "x": 50, "y": 51}, {
            "total": 2,
            "width": 50,
            "height": 50,
            "x": 313,
            "y": 50
        }]
    },
    imagePack: {
        "width": 400,
        "height": 202,
        "imageClassName": ".cm_mobHeader_artist-image--style-96-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003874107.jpg"
    }
};
var r400x267Image97 = {
    event: {"data": [{"total": 8, "width": 50, "height": 50, "x": 159, "y": 33}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-97-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003874501.jpg"
    }
};
var r400x274Image98 = {
    event: {
        "data": [{"total": 2, "width": 39, "height": 39, "x": 185, "y": 99}, {
            "total": 2,
            "width": 69,
            "height": 69,
            "x": 112,
            "y": 68
        }]
    },
    imagePack: {
        "width": 400,
        "height": 274,
        "imageClassName": ".cm_mobHeader_artist-image--style-98-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003877513.jpg"
    }
};
var r400x267Image100 = {
    event: {"data": [{"total": 1, "width": 39, "height": 39, "x": 177, "y": 57}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-100-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003879160.jpg"
    }
};
var r292x400Image101 = {
    event: {"data": [{"total": 3, "width": 39, "height": 39, "x": 112, "y": 37}]},
    imagePack: {
        "width": 292,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-101-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003879671.jpg"
    }
};
var r400x267Image102 = {
    event: {"data": [{"total": 3, "width": 39, "height": 39, "x": 200, "y": 71}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-102-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003879885.jpg"
    }
};
var r394x400Image103 = {
    event: {
        "data": [{"total": 4, "width": 39, "height": 39, "x": 176, "y": 26}, {
            "total": 4,
            "width": 39,
            "height": 39,
            "x": 299,
            "y": 131
        }, {"total": 4, "width": 39, "height": 39, "x": 45, "y": 151}]
    },
    imagePack: {
        "width": 394,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-103-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003888856.jpg"
    }
};
var r400x267Image104 = {
    event: {"data": [{"total": 3, "width": 39, "height": 39, "x": 176, "y": 80}]},
    imagePack: {
        "width": 400,
        "height": 267,
        "imageClassName": ".cm_mobHeader_artist-image--style-104-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003888881.jpg"
    }
};
var r250x250Image105 = {
    event: {"data": [{"total": 1, "width": 119, "height": 119, "x": 10, "y": 120}]},
    imagePack: {
        "width": 250,
        "height": 250,
        "imageClassName": ".cm_mobHeader_artist-image--style-105-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003891169.jpg"
    }
};
var r267x400Image106 = {
    event: {"data": [{"total": 1, "width": 61, "height": 61, "x": 90, "y": 50}]},
    imagePack: {
        "width": 267,
        "height": 400,
        "imageClassName": ".cm_mobHeader_artist-image--style-106-a",
        "elementContainingImage": {
            "0": {},
            "length": 1,
            "prevObject": {"0": {}, "length": 1},
            "selector": ".cm_mobHeader_artist_image",
            "containerProperties": {"width": 257, "height": 257}
        },
        "imgUrl": "../faces/js/assets/img/MI0003892823.jpg"
    }
};

module.exports = {
    r292x326Image0: r292x326Image0,
    r166x250Image1: r166x250Image1,
    r400x268Image3: r400x268Image3,
    r400x272Image5: r400x272Image5,
    r250x206Image4: r250x206Image4,
    r250x248Image7: r250x248Image7,
    r400x251Image6: r400x251Image6,
    r400x268Image8: r400x268Image8,
    r400x302Image9: r400x302Image9,
    r400x284Image11: r400x284Image11,
    r400x309Image13: r400x309Image13,
    r250x245Image14: r250x245Image14,
    r400x276Image17: r400x276Image17,
    r400x263Image18: r400x263Image18,
    r400x265Image19: r400x265Image19,
    r357x400Image20: r357x400Image20,
    r400x251Image21: r400x251Image21,
    r400x215Image22: r400x215Image22,
    r250x245Image25: r250x245Image25,
    r250x202Image26: r250x202Image26,
    r400x269Image28: r400x269Image28,
    r250x209Image27: r250x209Image27,
    r400x400Image31: r400x400Image31,
    r399x400Image34: r399x400Image34,
    r400x313Image35: r400x313Image35,
    r400x223Image36: r400x223Image36,
    r400x214Image38: r400x214Image38,
    r400x262Image41: r400x262Image41,
    r250x248Image43: r250x248Image43,
    r400x267Image45: r400x267Image45,
    r250x250Image49: r250x250Image49,
    r400x267Image50: r400x267Image50,
    r250x205Image52: r250x205Image52,
    r250x249Image55: r250x249Image55,
    r250x250Image54: r250x250Image54,
    r400x267Image56: r400x267Image56,
    r400x267Image58: r400x267Image58,
    r250x229Image59: r250x229Image59,
    r266x400Image62: r266x400Image62,
    r250x220Image64: r250x220Image64,
    r400x266Image65: r400x266Image65,
    r400x267Image66: r400x267Image66,
    r400x225Image67: r400x225Image67,
    r400x400Image69: r400x400Image69,
    r400x250Image70: r400x250Image70,
    r400x400Image71: r400x400Image71,
    r400x400Image75: r400x400Image75,
    r400x400Image76: r400x400Image76,
    r400x400Image77: r400x400Image77,
    r400x400Image78: r400x400Image78,
    r250x245Image79: r250x245Image79,
    r400x267Image80: r400x267Image80,
    r269x400Image81: r269x400Image81,
    r400x267Image82: r400x267Image82,
    r400x400Image84: r400x400Image84,
    r400x400Image85: r400x400Image85,
    r400x267Image86: r400x267Image86,
    r400x400Image87: r400x400Image87,
    r400x255Image89: r400x255Image89,
    r400x400Image90: r400x400Image90,
    r400x400Image93: r400x400Image93,
    r400x400Image94: r400x400Image94,
    r400x202Image96: r400x202Image96,
    r400x267Image97: r400x267Image97,
    r400x274Image98: r400x274Image98,
    r400x267Image100: r400x267Image100,
    r292x400Image101: r292x400Image101,
    r400x267Image102: r400x267Image102,
    r394x400Image103: r394x400Image103,
    r400x267Image104: r400x267Image104,
    r250x250Image105: r250x250Image105,
    r267x400Image106: r267x400Image106
};
